test/src/xquery/axes.xml

```diff
@@ -286,7 +286,7 @@
     <test output="text">
         <task>Preceding axis test: node()</task>
         <code>count(doc("/db/axes-test/axes1.xml")//line/word[. = 'working']/following::node())</code>
-        <expected>9</expected>
+        <expected>17</expected>
```

Comment:
Yes, you are right. I wondered why they were symmetric in number of nodes. So that is easily fixed. :)